### ISSUE TITLE
Enum value aliases error

### ISSUE THREAD

**Issue by amenzhinsky** [bufbuild/buf#195]
Hi there.

We had the following enum:

```protobuf
enum TacticsWidth {
  option allow_alias = true;
  TACTICS_WIDTH_UNSPECIFIED = 0;
  TACTICS_WIDTH_NARROW = 1;
  TACTICS_WIDTH_WIDTH_NORMAL = 2;
  TACTICS_WIDTH_WIDE = 3;
}
```

There was a typo in `TACTICS_WIDTH_WIDTH_NORMAL` value, so we decided to add an alias and deprecate it.

```protobuf
enum TacticsWidth {
  option allow_alias = true;
  TACTICS_WIDTH_UNSPECIFIED = 0;
  TACTICS_WIDTH_NARROW = 1;
  TACTICS_WIDTH_WIDTH_NORMAL = 2 [deprecated=true];
  TACTICS_WIDTH_NORMAL = 2;
  TACTICS_WIDTH_WIDE = 3;
}
```

But `buf check breaking` throws two errors:

```
foo.proto:735:32:Enum value "2" on enum "TacticsWidth" changed name from "TACTICS_WIDTH_WIDTH_NORMAL" to "TACTICS_WIDTH_NORMAL", "TACTICS_WIDTH_WIDTH_NORMAL".
foo.proto:736:26:Enum value "2" on enum "TacticsWidth" changed name from "TACTICS_WIDTH_WIDTH_NORMAL" to "TACTICS_WIDTH_NORMAL", "TACTICS_WIDTH_WIDTH_NORMAL".
```

I suppose it shouldn't do that because value is there, nothing should be broken.

**Comment by bufdev** [comment]
Yea that's fair - need to think about it a little bit more, but I think the breaking change detector should make sure that the new set covers the old set, not that it's equal. Thanks for the bug report.

**Comment by bufdev** [comment]
This is fixed in #197. This will go out in the next release. We'll update this issue when the next release goes out (likely Monday).

**Comment by bufdev** [comment]
This is released as v0.32.0.
